Dockerfile

```diff
@@ -33,26 +52,19 @@ ENV NTA_DATA_PATH /usr/local/src/nupic/prediction/data
 # OPF needs this
 ENV USER docker
 
-# Copy context into container file system
-ADD . $NUPIC
-
-WORKDIR /usr/local/src
-
-# Clone nupic.core
-RUN git clone `head -n 2 $NUPIC/.nupic_modules | tail -n 1 | sed -r "s/NUPIC_CORE_REMOTE = '(.+)'/\\1/"` && \
-    cd nupic.core && \
-    git reset --hard `head -n 3 $NUPIC/.nupic_modules | tail -n 1 | sed -r "s/NUPIC_CORE_COMMITISH = '(.+)'/\\1/"`
-
+# Set up nupic.core
+RUN pip install numpy pycapnp
```

Comment:
It depends on what the Dockerfile is for. Is it a base image for production applications? Or an easy to set up development environment? It's just unclear what the intent is.